modules/utils/2.0/utils.smk

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Snakefile Author:    Helena Winata, Bruno Grande
+# Module Author:                Bruno Grande
+# Additional Contributors:      N/A
+
+
+##### SETUP #####
+
+import sys, os
+from os.path import join
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+
+_UTILS = config["lcr-modules"]["utils"]
+LOG = "/logs/" + op._session.launched_fmt
+
+wildcard_constraints:
+    prefix = "[0-9]{2}-.*"
+
+
+##### RULES #####
+# _utils_bam_sort: Sort a BAM file using coordinates
+rule:
```

Comment:
Why do these rules not have rule names?

Reply:
I think it was to prevent multiple rules having the same name when the utils module is included multiple times. I'm not sure if that would be a problem anymore. I'll try adding the names and see if it causes problems